src/helm/benchmark/model_deployment_registry.py

```diff
@@ -104,5 +134,152 @@ def maybe_register_model_deployments_from_base_path(base_path: str) -> None:
+        ]
+        if len(non_deprecated_deployments) > 0:
+            chosen_deployment = non_deprecated_deployments[0]
+        # There are no non-deprecated deployments, so there are two options:
+        # 1. If we can return an empty string, return it. (no model deployment is available)
+        # 2. If we can't return an empty string, return the first deployment (even if it's deprecated).
+        elif can_return_empy:
+            return ""
+        else:
+            hlog(f"WARNING: All model deployments for model {model_arg} are deprecated.")
+            chosen_deployment = available_deployments[0]
+        if warn_arg_deprecated:
+            hlog(
+                f"Choosing {chosen_deployment.name} (the first one) as "
+                f"the default model deployment for model {model_arg}"
+            )
+            hlog("If you want to use a different model deployment, please specify it explicitly.")
+        return chosen_deployment.name
+
+    # Some models are added but have no deployments yet.
+    # In this case, we return an empty string if can_return_empy is True, otherwise we raise an error.
+    if can_return_empy:
+        return ""
+    raise ValueError(f"Model deployment {model_arg} not found")
+
+
+def get_model_deployments_by_host_organization(host_organization: str) -> List[str]:
+    """
+    Gets models by host organization.
+    Example:   together => [" together/bloom", "together/t0pp", ...]
+    """
+    register_deployments_if_not_already_registered()
+    return [
+        deployment.name for deployment in ALL_MODEL_DEPLOYMENTS if deployment.host_organization == host_organization
+    ]
+
+
+def get_model_deployment_host_organization(name: str) -> str:
+    """
+    Extracts the host organization from the model deployment name.
+    Example: "huggingface/t5-11b" => "huggingface"
+    """
+    deployment: ModelDeployment = get_model_deployment(name)
+    return deployment.host_organization
+
+
+def get_metadata_for_deployment(deployment_name: str) -> ModelMetadata:
+    """
+    Given a deployment name, returns the corresponding model metadata.
+    """
+    deployment: ModelDeployment = get_model_deployment(deployment_name)
+    return get_model_metadata(deployment.model_name or deployment.name)
+
+
+def get_model_names_with_tokenizer(tokenizer_name: str) -> List[str]:
+    """Get all the name of the models with tokenizer `tokenizer_name`."""
+    register_deployments_if_not_already_registered()
+    deployments: List[ModelDeployment] = [
+        deployment for deployment in ALL_MODEL_DEPLOYMENTS if deployment.tokenizer_name == tokenizer_name
+    ]
+    return [deployment.model_name or deployment.name for deployment in deployments]
+
+
+def register_deployments_if_not_already_registered() -> None:
+    global DEPLOYMENTS_REGISTERED
+    if not DEPLOYMENTS_REGISTERED:
+        maybe_register_model_deployments_from_base_path(CONFIG_PATH)
+        DEPLOYMENTS_REGISTERED = True
+
+
+def maybe_register_helm():
```

Comment:
`register_helm_configurations`? I think this should be in another file, otherwise you end up with unnecessary inter-file dependencies.